microprofile/testing/testng/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>helidon-microprofile-cdi</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>provided</scope>
```

Comment:
Again, provided scope is not good here.
We expect TestNG (in this case), and Helidon as provided, as these must be present to even use this module, and users add them to their projects for testing.
Mockito is not an auto-include for testing, so we should not do this.